applications/mne_analyze/libs/anShared/Model/ecdsetmodel.cpp

```diff
@@ -179,6 +181,19 @@ bool EcdSetModel::setData(const QModelIndex &index, const QVariant &value, int r
 }
 
 
+//*************************************************************************************************************
+
+bool EcdSetModel::saveToFile()
+{
+    if(m_modelPath.sDirectoryPath == ECD_SET_MODEL_DEFAULT_DIR_PATH) {
```

Comment:
Having a bit of trouble understanding this, maybe a short comment will explain the directory check.